src/activities/GetTimeZone.ts

```diff
@@ -0,0 +1,25 @@
+import type { IActivityHandler } from "@geocortex/workflow/runtime";
+
+interface GetTimeZoneOutputs {
+    /**
+     * @description The time zone of the end user.
+     */
+    result: string;
+}
+
+/**
+ * @displayName Get Time Zone
+ * @defaultName timeZone
+ * @category Web APIs
+ * @description Gets time zone of the end user as reported by the web browser.
```

Comment:
Gets **the** time zone of the end user as reported by the web browser.

Reply:
Fixed